extensions/3rdparty/LyricWiki/BetterLyricWikiTextSnippets.php

```diff
@@ -22,7 +22,6 @@
 $dir = dirname(__FILE__);
 
 // Internalization
```

Comment:
What is _internalization_, though?  Is it a process or an act of making something _internal_? :-)